vendor/processors/stripe_processor.py

```diff
@@ -198,6 +268,197 @@ def build_subscription(self, subscription, payment_method_id):
             'default_payment_method': payment_method_id,
             'metadata': {'site': self.invoice.site}
         }
+    
+    def build_subscription(self, customer_id, items, payment_id, offer):
+        subscription_data = {
+            'customer': customer_id,
+            'items': [{'price': item.id} for item in items],
+            'default_payment_method': payment_id,
+            'trial_period_days': None if offer.term_details['trial_days'] < 1 else offer.term_details['trial_days'],
+            'metadata': {'site': offer.site.pk}
+
+        }
+
+        subscription = self.stripe_create_object(self.stripe.Subscription, subscription_data)
+        
+        return subscription
+
+    ##########
+    # Stripe & Vendor Objects Synchronization
+    ##########
+
+    def get_stripe_customers(self):
+        """
+        Returns all Stripe created customers
+        """
+        query = self.build_search_query([{
+            'key_name': 'site',
+            'key_value': self.site.pk,
+            'field_type': 'metadata'
+        }])
+        customer_search = self.stripe_query_object(self.stripe.Customer, {'query': query})
+        if customer_search:
+            return customer_search['data']
+        return None
```

Comment:
Add an empty line between if and `customer_search` also between the return in the if and the one next to it